index.html

```diff
@@ -373,6 +373,29 @@ <h2>
             NO
           </td>
         </tr>
+        <tr data-cite="window-placement">
+          <td class="string-token">
+            "[=window-management=]"
+          </td>
+          <td class="is-policy-controlled">
+            YES
```

Comment:
I assume https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/permissions_policy/permissions_policy_features.json5;l=410?q=permissions_policy_features.json5 will eventually be updated to the new name, but confirming Chromium has a feature policy for this.

Reply:
Yes we are in the midst of renaming the permission & policy name from "window-placement" to "window-management" tracked in https://github.com/w3c/window-placement/issues/114 which has the associated chromium bug (https://crbug.com/1328581).